ui-lib/src/main/res/ui/advanced_settings/values/strings.xml

```diff
@@ -1,11 +1,10 @@
 <resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="advanced_settings_backup_wallet">Recovery phrase</string>
-    <string name="advanced_settings_export_private_data">Export private data</string>
-    <string name="advanced_settings_choose_server">Choose a server</string>
+    <string name="advanced_settings_title">ADVANCED SETTINGS</string>
```

Comment:
Please ensure that the capitalization is done programmatically rather than just relying on the string value.

Reply:
done